zerver/context_processors.py

```diff
@@ -135,6 +135,12 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
         settings_path = "/etc/zulip/settings.py"
         settings_comments_path = "/etc/zulip/settings.py"
 
+    self_hosted_signup_page = False
+    if (
+        settings.DEVELOPMENT or not realm or (realm.plan_type == Realm.PLAN_TYPE_SELF_HOSTED)
```

Comment:
Not sure about this.

Reply:
We should always use `realm is not None` over `not realm`, for readability -- generally Python falsiness isn't a feature we want to use.

Do we need `settings.DEVELOPMENT`? I thought the development environment had `realm.plan_type == Realm.PLAN_TYPE_SELF_HOSTED`.

Reply:
Testing removing the `settings.DEVELOPMENT` check, I think that works fine.